internal/config/instancetypes/azure.go

```diff
@@ -43,6 +43,40 @@ var AzureCVMInstanceTypes = []string{
 	"Standard_EC48ads_v5",
 	"Standard_EC64ads_v5",
 	"Standard_EC96ads_v5",
+
+	// TDX CVMs
+	// DCesv5-series
+	"Standard_DC4es_v5",
```

Comment:
Create a new variable for this so we can more easily differentiate between SEV and TDX instances